src/options.cpp

```diff
@@ -761,13 +761,6 @@ void initOptions()
                                  );
 
     ////////////////////////////GRAPHICS/////////////////////////
-    OPTIONS["NO_BRIGHT_BACKGROUNDS"] = cOpt("graphics", _("No bright backgrounds"),
-                                            _("If true, bright backgrounds are not used - some consoles are not compatible."),
-                                            false
```

Comment:
Hmm.  Just for safety's sake, how will this affect bright-background-incompatible consoles--will they be able to access the menu and enable the template (and then later modify it, in the event that the user happens to need/want modified colors)?